lib/pages/auth/forgot_password.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:chatme/pages/auth/login_page.dart';
+import 'package:chatme/widgets/common_widgets.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+
+class ForgotPassword extends StatefulWidget {
+  const ForgotPassword({super.key});
+
+  @override
+  State<ForgotPassword> createState() => _ForgotPasswordState();
+}
+
+class _ForgotPasswordState extends State<ForgotPassword> {
+  final formKey = GlobalKey<FormState>();
+  AutovalidateMode autovalidate = AutovalidateMode.disabled;
+  final emailController = TextEditingController();
+
+  @override
+  void dispose() {
+    emailController.dispose();
+    super.dispose();
+  }
+
+  void showSnackBar(String text) {
+    ScaffoldMessenger.of(context).showSnackBar(
+      SnackBar(
+          content: Text(
+            text,
+            style: const TextStyle(
+              color: Colors.white,
+              fontSize: 15,
+              fontWeight: FontWeight.w600,
+            ),
+          ),
+          backgroundColor: Colors.blue),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+          appBar: AppBar(backgroundColor: Theme.of(context).primaryColor,
+            leading: IconButton(
+              icon: const Icon(Icons.arrow_back),
+              onPressed: () {
+                nextScreen(context, const LoginPage());
+              },
+            ),
+          ),
+          body: Center(
+            child: Padding(
+              padding: const EdgeInsets.fromLTRB(0, 110, 0, 0),
+              child: Form(
+                  key: formKey,
+                  child: Column(
+                    children: [
+                      const Padding(
+                        padding: EdgeInsets.all(50.0),
+                        child: Text(
+                          'Receive an Email to Reset your Password',
+                          style: TextStyle(
+                            fontSize: 22,
+                            fontWeight: FontWeight.bold,
+                            color: Colors.black,
+                          ),
+                        ),
+                      ),
+                      Padding(
```

Comment:
Hi mounika,
  small suggestion, In padding, try to give all equal sides. Please do it and sent it to me.

Reply:
Thank you Divya, for your suggestions. I will done it soon as  possible.